### DISCUSSION TITLE
更新0.7.3版本后图片显示样式改版了，能否恢复到以前的样式

### DISCUSSION THREAD

**Discussion by leoneoo** [usememos/memos#497] [Q&A] [answered]
<img src="https://oneoo-1253919472.cos.ap-shanghai.myqcloud.com/oneooimg/old.png" alt="image" style="max-width: 100%;">

<img src="https://oneoo-1253919472.cos.ap-shanghai.myqcloud.com/oneooimg/0.73%E7%89%88%E6%9C%AC.png" alt="image" style="max-width: 100%;">

**Comment by leoneoo** [comment]
0.7.3版本插入图片后右侧类似多了滚动条，但是貌似影响了整体美观！另外能否增加一个图片放大的功能，并可实现多倍放大。

**Comment by boojack** [comment, marked as answer]
> 图片右侧类似多了滚动条

已修复，将会在下个发布中更新

> 能否增加一个图片放大的功能，并可实现多倍放大

可以新建一个 feature request 的 issue，而不是一个 discussion。
